src/mammos_entity/_base.py

```diff
@@ -131,7 +133,9 @@ def __init__(
         | mammos_units.Quantity
         | mammos_entity.Entity = 0,
         unit: str | None | mammos_units.UnitBase = None,
+        description: str | None = None,
     ):
+        self.description = description if description is not None else ""
```

Comment:
Could you please make `description` a property to (i) check the type on assignment and (ii) have it documented better.